File: customers-staff-master/src/CustomersAndStaff/FrontApi.Dto/Calendars/ServiceCalendarRecordInfoDto.cs
using System;

using Market.CustomersAndStaff.Models.Common;

namespace Market.CustomersAndStaff.FrontApi.Dto.Calendars
{
    public class ServiceCalendarRecordInfoDto
    {
        public Guid? CustomerId { get; set; }
        public Guid[] ProductIds { get; set; }
        public TimePeriod Period { get; set; }
        public string Comment { get; set; }
        public bool IsOnlineRecord { get; set; }
    }

    public enum RecordStatusDto
    {
        Active,
        Canceled,
        Removed,
    }

    public enum CustomerStatusDto
    {
        Active,
        ActiveAccepted,
        Completed,
        CanceledBeforeEvent,
        NotCome,
        NoService,
        Mistake,
    }
}

File: customers-staff-master/src/Front/src/api/new/dto/ServiceCalendar/ServiceCalendarRecordInfoDto.ts
import { Guid } from "../Guid";
import { TimePeriodDto } from "../WorkingCalendar/WorkersScheduleDtos";

export interface ServiceCalendarRecordInfoDto {
  customerId?: Guid;
  productIds: Guid[];
  period: TimePeriodDto;
  comment: string;
}

export enum CustomerStatusDto {
  Active = "active",
  ActiveAccepted = "activeAccepted",
  Completed = "completed",
  CanceledBeforeEvent = "canceledBeforeEvent",
  NotCome = "notCome",
  NoService = "noService",
  Mistake = "mistake",
}

export enum RecordStatusDto {
  Active = "active",
  Canceled = "canceled",
  Removed = "removed",
}
